pkg/converter/convert_test.go

```diff
@@ -1042,6 +1044,26 @@ func TestConvert(t *testing.T) {
 			},
 			command: execName,
 		},
+		{
+			name:        "convert with context specified, auth info not specified by the context should not be changed",
```

Comment:
maybe we should also add a test case for context specified, auth info not found case to ensure the behavior

Reply:
i think what you are suggesting is the essence of this test case? for any unmatched auth info, they should remain as is